packages/core/src/common/_mixins.scss

```diff
@@ -99,30 +99,8 @@ $pt-dark-intent-text-colors: (
   @return $number;
 }
 
-@mixin force-hardware-acceleration() {
```

Comment:
this is a slight break for any Blueprint consumer who's reaching in to import `@blueprintjs/core/src/common/mixins` in Sass code, but that's not really part of our public API and a search across GitHub reveals that the break won't have any real impact, so I think it's ok.